### DISCUSSION TITLE
Thanks, success scanning on Canon PIXMA G4010 series

### DISCUSSION THREAD

**Discussion by kateamon** [alexpevzner/sane-airscan#237] [General]
Today (May 30, 2022) success scanning WSD mode on ESCL Canon G4010 series flatbed scanner. Sharing the good news.

**Comment by alexpevzner** [comment]
Hi, @kateamon,

thank you for your feedback!

I still want to clarify for my records, what protocol do you actually use, WSD or ESCL. Output of the `scanimage -L` command will help.
